### ISSUE TITLE
[BUG] Crash when clearing failed uploads

### ISSUE THREAD

**Issue by davigonz** [owncloud/android#2568]
Similar to https://github.com/owncloud/android/issues/2563

### Actual behaviour
- App crashes when clearing failed uploads.

### Expected behaviour
- App should behave normally when clearing failed uploads.
 
### Steps to reproduce
1. Upload a file.
2. Turn off wifi to make the upload fail.
2. Go to uploads view.
3. Tap three dots button and press "Clear failed".

### Environment data
Android version: 7.1.2

Device model: Nexus 6P

Stock or customized system: Stock

ownCloud app version: 2.11 beta

ownCloud server version: 10.0.10

#### ownCloud log (data/owncloud.log)
```
2019-06-06 09:26:05.137 15480-15480/com.owncloud.android.debug E/AndroidRuntime: FATAL EXCEPTION: main
    Process: com.owncloud.android.debug, PID: 15480
    java.lang.IllegalArgumentException: Selection not allowed, use parameterized queries
        at com.owncloud.android.providers.FileContentProvider.delete(FileContentProvider.java:272)
        at com.owncloud.android.providers.FileContentProvider.delete(FileContentProvider.java:261)
        at android.content.ContentProvider$Transport.delete(ContentProvider.java:340)
        at android.content.ContentResolver.delete(ContentResolver.java:1373)
        at com.owncloud.android.datamodel.UploadsStorageManager.clearFailedButNotDelayedForWifiUploads(UploadsStorageManager.java:429)
        at com.owncloud.android.ui.activity.UploadListActivity.onOptionsItemSelected(UploadListActivity.java:206)
        at android.app.Activity.onMenuItemSelected(Activity.java:3207)
        at androidx.fragment.app.FragmentActivity.onMenuItemSelected(FragmentActivity.java:436)
        at androidx.appcompat.app.AppCompatActivity.onMenuItemSelected(AppCompatActivity.java:196)
        at androidx.appcompat.view.WindowCallbackWrapper.onMenuItemSelected(WindowCallbackWrapper.java:109)
        at androidx.appcompat.view.WindowCallbackWrapper.onMenuItemSelected(WindowCallbackWrapper.java:109)
        at androidx.appcompat.app.ToolbarActionBar$2.onMenuItemClick(ToolbarActionBar.java:64)
        at androidx.appcompat.widget.Toolbar$1.onMenuItemClick(Toolbar.java:204)
        at androidx.appcompat.widget.ActionMenuView$MenuBuilderCallback.onMenuItemSelected(ActionMenuView.java:781)
        at androidx.appcompat.view.menu.MenuBuilder.dispatchMenuItemSelected(MenuBuilder.java:840)
        at androidx.appcompat.view.menu.MenuItemImpl.invoke(MenuItemImpl.java:158)
        at androidx.appcompat.view.menu.MenuBuilder.performItemAction(MenuBuilder.java:991)
        at androidx.appcompat.view.menu.MenuPopup.onItemClick(MenuPopup.java:128)
        at android.widget.AdapterView.performItemClick(AdapterView.java:310)
        at android.widget.AbsListView.performItemClick(AbsListView.java:1164)
        at android.widget.AbsListView$PerformClick.run(AbsListView.java:3132)
        at android.widget.AbsListView$3.run(AbsListView.java:4047)
        at android.os.Handler.handleCallback(Handler.java:751)
        at android.os.Handler.dispatchMessage(Handler.java:95)
        at android.os.Looper.loop(Looper.java:154)
        at android.app.ActivityThread.main(ActivityThread.java:6121)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:889)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:779)
```

### PR

App

**Comment by sd1998** [comment]
@davigonz I would like to work on this.

**Comment by davigonz** [comment]
>  I would like to work on this.

Sure, let's fix it via https://github.com/owncloud/android/pull/2566, thanks!

**Comment by davigonz** [comment]
Done, to be included in 2.11
